Can't compile JS templates

**Godot version:**
0c86c1ad152d2b523dc125eb79e838563b3d5f69

**OS/device including version:**
`Linux 4.14.8-300.fc27.x86_64`

**Issue description:**
```bash
$ scons platform=javascript tools=no target=release_debug
scons: Reading SConscript files ...
WebM SIMD optimizations are disabled. Check if your CPU architecture, CPU bits or platform are supported!
Checking for C header file mntent.h... (cached) no
scons: done reading SConscript files.
scons: Building targets ...
[Initial build] Compiling ==> platform/javascript/javascript_main.cpp
platform/javascript/javascript_main.cpp:73:11: error: allocating an object of abstract class type 'OS_JavaScript'
        os = new OS_JavaScript(argv[0], NULL);
                 ^
core/os/os.h:329:15: note: unimplemented pure virtual method 'set_custom_mouse_cursor' in 'OS_JavaScript'
        virtual void set_custom_mouse_cursor(const RES &p_cursor, CursorShape p_shape, const Vector2 &p_hotspot) = 0;
                     ^
1 error generated.
ERROR:root:compiler frontend failed to generate LLVM bitcode, halting
scons: *** [platform/javascript/javascript_main.javascript.opt.debug.bc] Error 1
scons: building terminated because of errors.

```

**Steps to reproduce:**
```bash
$ scons platform=javascript tools=no target=release_debug
# Or:
$ scons platform=javascript tools=no target=release
```

--- a/platform/javascript/os_javascript.cpp
+++ b/platform/javascript/os_javascript.cpp
@@ -782,6 +782,9 @@ void OS_JavaScript::set_cursor_shape(CursorShape p_shape) {
 		set_css_cursor(godot2dom_cursor(cursor_shape));
 }
 
+void OS_JavaScript::set_custom_mouse_cursor(const RES &p_cursor, CursorShape p_shape, const Vector2 &p_hotspot) {
+}
+
 void OS_JavaScript::main_loop_begin() {
 
 	if (main_loop)

--- a/platform/javascript/os_javascript.h
+++ b/platform/javascript/os_javascript.h
@@ -128,6 +128,7 @@ public:
 	virtual bool is_userfs_persistent() const;
 
 	virtual void set_cursor_shape(CursorShape p_shape);
+	virtual void set_custom_mouse_cursor(const RES &p_cursor, CursorShape p_shape, const Vector2 &p_hotspot);
 
 	void main_loop_begin();
 	bool main_loop_iterate();

